doc/changes.txt

```diff
@@ -42,6 +42,7 @@ Improvements:
 - Added a @c USE_COLORED_OUTPUT option to CMake to display compiler output
   in color.
 - Added an @c OPENVDB_CODE_COVERAGE option to CMake.
+- Marked VFX 17 platform version dependencies as deprecated.
```

Comment:
Maybe change this to "CMake now issues deprecation warnings for 2017 VFX Reference Platform version dependencies.  In particular, C++11-only compilers are now deprecated; OpenVDB 7.0 will require a C++14-compatible compiler."

Reply:
Thanks, done